index.qmd

```diff
@@ -15,5 +15,115 @@ execute:
 editor: source
 ---
 
-## Placeholder slide
+```{r}
+#| include: false
+
+# setup
+pak::pkg_install("epidatr")
+library(epidatr)
+library(dplyr)
+```
+
+## What is `epidatr`?
+
+The [Delphi Epidatr package](https://cmu-delphi.github.io/epidatr/) is an R front-end for the [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/), which provides real-time access to epidemiological surveillance data for influenza, COVID-19, and other diseases for the USA at various geographical resolutions, both from official government sources such as the [Center for Disease Control (CDC)](https://www.cdc.gov/datastatistics/index.html) and [Google Trends](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/google-symptoms.html) and private partners such as [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Change Healthcare](https://www.changehealthcare.com/). It is built and maintained by the Carnegie Mellon University [Delphi research group](https://delphi.cmu.edu/).
+
+This package is designed to streamline the downloading and usage of data from the [Delphi Epidata
+API](https://cmu-delphi.github.io/delphi-epidata/). It provides a simple R interface to the API, including functions for downloading data, parsing the results, and converting the data into a tidy format. The API stores a historical record of all data, including corrections and updates, which is particularly useful for accurately backtesting forecasting models. We also provide packages for downstream data processing ([epiprocess](https://github.com/cmu-delphi/epiprocess)) and modeling ([epipredict](https://github.com/cmu-delphi/epipredict)).
+
+
+## Installation
+
+You can install the stable version of this package from CRAN:
+
+```{r}
+#| eval: false
+#| echo: true
+install.packages("epidatr")
+pak::pkg_install("epidatr")
+renv::install("epidatr")
+```
+
+Or if you want the development version, install from GitHub:
+
+```{r}
+#| eval: false
+#| echo: true
+# Install the dev version using `pak` or `remotes`
+pak::pkg_install("cmu-delphi/epidatr")
+remotes::install_github("cmu-delphi/epidatr")
+renv::install("cmu-delphi/epidatr")
+```
+
+## API key setup
+
+The Delphi API requires a (free) API key for full functionality. To generate
+your key, register for a pseudo-anonymous account
+[here](https://api.delphi.cmu.edu/epidata/admin/registration_form) and see more
+discussion on the [general API
+website](https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html). See the
+`set_api_key()` function documentation for details on how to use your API key.
```

Comment:
I copied this from the readme on `dev`, but realized that `set_api_key` isn't available in the CRAN version of epidatr. We probably don't want to give the [old API key advice](https://github.com/cmu-delphi/epidatr/blob/0adb796a54aee22cc54f9e1d0a724492892267b9/vignettes/epidatr.Rmd#L31-L33), though.

Reply:
Right, I'm thinking we just give the new advice for `save_api_key()`. Hopefully we can get it to CRAN before Tuesday and if not, we'll get it soon enough after the meeting. Really eager users can probably figure out how to use environment variables on their own.

Reply:
Sounds good, thanks!